modules/openapi-generator/src/main/resources/cpp-rest-sdk-client/apiclient-header.mustache

```diff
@@ -32,6 +33,9 @@ public:
     ApiClient( std::shared_ptr<ApiConfiguration> configuration = nullptr );
     virtual ~ApiClient();
 
+    const ResponseHeadersHandlerType& getResponseHeadersHandler() const;
```

Comment:
I cannot find this typedef, where is it?

Reply:
@etherealjoy your are right, I forgot that. Now it is fixed.